### PR TITLE
Updated volumetype param description

### PR DESCRIPTION
**Pull request by msmbaldwin**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md [hunk 1]
```diff
@@ -565,10 +567,11 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Specifies the type of virtual machine volumes on which to perform the encryption operation.
-Allowed values for Windows virtual machines: All, OS, and Data.
-The only allowed value for Linux virtual machines: Data.
-The VolumeType parameter is required when encrypting Linux virtual machines, but can be omitted when encrypting Windows virtual machines.
+Specifies the type of virtual machine volumes on which to perform encryption operation: OS, Data, or All. If this parameter is omitted, the operation defaults to All.
```
Comment (jackrichins) [inline src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md]: Should shorten this to not specify a default as the following sentences clarify default for Windows and Linux as has no default.

File: src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md [hunk 2]
```diff
@@ -274,7 +274,12 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Type of the volume (OS or Data) on which to perform encryption operation. This parameter is required when encrypting Linux virtual machines, and must be set to a value supported by the Linux distribution. This parameter can be omitted when encrypting Windows virtual machines.
+Specifies the type of virtual machine volumes on which to perform encryption operation: OS, Data, or All. If this parameter is omitted, the operation defaults to All.
```
Comment (jackrichins) [inline src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md]: No default for Linux, just Windows.

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md (hunk 1 above)
Comment (msmbaldwin) [reply src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md]: Removed the final sentence.

File: src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md (hunk 2 above)
Comment (msmbaldwin) [reply src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md]: Removed the final sentence, moved default into to Windows only.
